app/views/content/is-teaching-right-for-me/engineers-teach-physics/_article.html.erb

```diff
@@ -101,28 +57,11 @@
     </p>
   </section>
 
-  <section class="col col-640 col-space-m">
-    <%= render Content::QuoteComponent.new(
-      text: "My engineering degree provided a natural transition to teaching physics and
-        I value the way science – especially physics – teaches a clear thought process
-        and a way to make rational sense of the world.",
-      name: "Roger, physics teacher",
-      large: true
-    ) %>
-  </section>
-
   <section class="col col-720 col-space-m">
-    <h2>Fund your teacher training</h2>
-    <p>
-      Scholarships of <%= v :scholarships_physics %> or bursaries of <%= v :bursaries_postgraduate_physics %> are available for trainee physics teachers starting their teacher training course between September 2024 and July 2025.
-    </p>
-
-    <p>
-      <a href="/funding-and-support/scholarships-and-bursaries">Find out about your eligibility for bursaries and scholarships</a>.
-    </p>
-
-    <p>
-      You can also <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/">search for physics teacher training courses</a> that are not part of the Engineers teach physics training programme.
+    <h3>Fund your teacher training</h3>
+    <p>Scholarships of <%= v :scholarships_physics %> or bursaries of <%= v :bursaries_postgraduate_physics %> are available for trainee physics teachers starting their teacher training course between September 2024 and July 2025.</p>
```

Comment:
```suggestion
    <p>Scholarships of <%= v :scholarships_physics %> or bursaries of <%= v :bursaries_postgraduate_physics %> are available for eligible trainee physics teachers.</p>
```

I suggest we remove the dates in line with other subject pages, to make future maintenance easier.